tests/api/task1.test.js

```diff
@@ -0,0 +1,43 @@
+import {test, expect} from '@playwright/test';
+
+test.describe("API testing", () => {
+
+  const baseUrl = "http://localhost:3030/"
+
+  test("Get all Lists with their List Items", async ({request}) => {
+
+    const response = await request.get(`${baseUrl}/lists`)
+    expect(response.status()).toBe(200)
+
+    const responseBody = JSON.parse(await response.text())
+    console.log(responseBody)
+  })
+
+  test("Assert Invalid Endpoint for GET request", async ({request}) => {
+
+    const response = await request.get(`${baseUrl}/invalidendpoint`)
+    expect(response.status()).toBe(404)
+  })
+
+  test("Add a List with POST request", async ({request}) => {
+
+    const response = await request.post(`${baseUrl}/lists/add`, {
+      data: {
+        name: "Toptal"
+      },
+    })
+
+      const responseBody = JSON.parse(await response.text())
+    // console.log(responseBody)
+      expect(responseBody.name).toBe("Toptal")
+      expect(responseBody.items).toBeTruthy()
+  })
+
+    test("Delete a List", async ({request}) => {
+
+      const response = await request.delete(`${baseUrl}/lists/4`)
```

Comment:
JFYI, this test uses a hardcoded list id. In case there is no such id in the database or it's already deleted, the result of the test can be unpredictable. One of the possible approaches can be the following:
1. Create a new list. Get the response and save newly created list_id into a variable.
2. Make a delete call using the variable, e.g. `const response = await request.delete(`${baseUrl}/lists/${listId}`)`
As I remember, specifically in this app `POST` request does not return listId so we can leave this test as it is.